scripts/choose_backend.lua

```diff
@@ -0,0 +1,19 @@
+core.Debug("Loading lua code for routing to the right backend")
```

Comment:
Why another programming language? Although it's pretty readable, I don't think we need to support more.

Reply:
I didn't choose it, that's what HAproxy works with...